Tests/js/views/tabs/assets/ez-locationviewversionstabview-tests.js

```diff
@@ -282,109 +282,6 @@ YUI.add('ez-locationviewversionstabview-tests', function (Y) {
         },
     });
 
-    fireCreateDraftEventTest = new Y.Test.Case({
```

Comment:
you forgot to remove the variable declaration and the `Y.Test.Runner.add(fireCreateDraftEventTest);` call below